docs/user.md

```diff
@@ -32,15 +32,24 @@ Table Of Contents:
 
 ## Process Overview <a name="process"></a>
 
-More detail on each step can be found in the sections further below.
+This section describes the steps of the process when submiting the results to Red Hat Partner,
+if the goal is not sharing the results to Red Hat, you can go to the next section.
+
+Overview of the process to apply the results to the Red Hat Partner support case:
 
-1. Prepare the OpenShift cluster to be certified
-2. Download and install the provider certification tool
-3. Run provider certification tool
+0. Install an OpenShift cluster on **the version desired** to be validated
+1. Prepare the OpenShift cluster to run the validated environment
+2. Download and install the OPCT
+3. Run the OPCT
 4. Monitor tests 
 5. Gather results
-6. Share certification results and [must gather](https://docs.openshift.com/container-platform/latest/support/gathering-cluster-data.html) with Red Hat
+6. Share the results archive with Red Hat Partner support case
+
+You may be asked by Support Engineers to repeat the process a few times depending on the results.
 
+Finally, you will be asked to manually upgrade the cluster to the next Y-release.
```

Comment:
Y-release may be an uncommon nomenclature outside of Red Hat.  You might consider 'minor release'.

Reply:
Thanks! Ptal?